build.sbt

```diff
@@ -1212,7 +1224,9 @@ lazy val runtime = (project in file("engine/runtime"))
       s"--upgrade-module-path=${file("engine/runtime/build-cache/truffle-api.jar").absolutePath}"
     ),
     Test / fork := true,
-    Test / envVars ++= distributionEnvironmentOverrides ++ Map("ENSO_TEST_DISABLE_IR_CACHE" -> "false"),
+    Test / envVars ++= distributionEnvironmentOverrides ++ Map(
```

Comment:
There is so many formatting changes! If they really have to be done, could they be performed in a separate commit (that doesn't need a review), please?

Reply:
Ok, I split the commit into two, the changes sans formatting can be seen here: https://github.com/enso-org/enso/pull/3450/files/0f942bb39be3ccfe33a4758b42a673f9e17a2ee8..17b867a1cec5f6d0ba8ca6d27e6e727b06fdf493

(just the files view with the first commit of the PR excluded)